iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -219,13 +206,14 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
                 .filter(key -> !metadata.properties().containsKey(key))
                 .collect(Collectors.toSet());
       }
-
-      Map<String, String> summary =
-          Optional.ofNullable(metadata.currentSnapshot())
-              .map(Snapshot::summary)
-              .orElseGet(ImmutableMap::of);
-      setHmsTableParameters(
-          newMetadataLocation, tbl, metadata, removedProps, hiveEngineEnabled, summary);
+      HMSTablePropertyHelper.setHmsTableParameters(
+          newMetadataLocation,
+          tbl,
+          metadata,
+          removedProps,
+          hiveEngineEnabled,
+          maxHiveTablePropertySize,
+          currentMetadataLocation());
```

Comment:
Can you format it horizontally?

Reply:
No, iceberg code formatter wants it this way...

Reply:
I built Hive after having formatted it horizontally on my local and it was successful.

```
      HMSTablePropertyHelper.setHmsTableParameters(newMetadataLocation, tbl, metadata, removedProps, hiveEngineEnabled,
          maxHiveTablePropertySize, currentMetadataLocation());
```

Reply:
For me it failed, and wanted to have it that way. I didn't wanted to have it that way.